Taskwarrior Pomodoro/AppDelegate.swift

```diff
@@ -367,11 +472,12 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
     func startTaskById(taskId: String) {
         activeTaskId = taskId
         
-        let task = NSTask()
-        task.launchPath = taskPath
-        task.arguments = [taskId, "start"]
-        task.launch()
-        task.waitUntilExit()
+        taskCommand([taskId, "start"])
+        activeTaskPomodorosLogUUID = currentPomodorosLogUUID
```

Comment:
`currentPomodorosLogUUID` holds uuid from last refresh (every time menu is pulled down)
`activeTaskPomodorosLogUUID` takes and holds that uuid when task is activated to log pomodoro when task is done.

It's a matter of decision if a pomodoro that started one day and finished the next day should be logged at a day it started or at a day it finished.